src/Slices/CreateInstance/UI/CreateInstance.test.tsx

```diff
@@ -204,7 +204,7 @@ test("Given the CreateInstance View When creating an instance with Inter-service
   });
   expect(apiHelper.pendingRequests[0]).toEqual({
     method: "GET",
-    url: `/lsm/v1/service_inventory/${InterServiceRelations.editable.entity_type}?include_deployment_progress=False&limit=100&filter.order_id=a`,
+    url: `/lsm/v1/service_inventory/${InterServiceRelations.editable.entity_type}?include_deployment_progress=False&limit=250&filter.order_id=a`,
```

Comment:
why did you change the default here to 250?

Reply:
This request is for AutoComplete functionality, it was using the max range available, to serve the most possibilities  to autocomplete so I thought that it should be increased from previous max to current